### DISCUSSION TITLE
How to generate bitstream for a given FPGA architecture

### DISCUSSION THREAD

**Discussion by Sun1927** [lnis-uofu/OpenFPGA#413] [Q&A] [answered]
Hi,

We are trying to use Openfpga to generate bitstream for our own FPGA architecture but before that, it would be very helpful if there is any bitstream generation example for the existing FPGA architectures in Openfpga(such as k4_frac_N4_40nm in vpr_arch) so that we can have a basic understanding of the bitstream generation flow in Openfpga.

Thanks for reading.

**Comment by tangxifan** [comment, marked as answer]
Hi @Sun1927 
  Please checkout these examples:
  https://github.com/lnis-uofu/OpenFPGA/tree/master/openfpga_flow/tasks/fpga_bitstream/generate_bitstream
